app/controllers/to_do_items_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ToDoItemsController < ApplicationController
+  def create
+    list = List.find params[:id]
+    @item = list.items.new item_params
+
+    @result = @item.save
+    @current_item = @item
```

Comment:
@vlasiak, regarding `@result` variable. Let's base our solutions on objects, not a set of variables. I suggest to use item, for example you can check [whether it is saved](http://apidock.com/rails/v4.2.1/ActiveRecord/Persistence/persisted%3F).
So please correct and remove usage of `@result` variable.